lib/saml/user.rb

```diff
@@ -81,27 +81,22 @@ def authn_context
       raise
     end
 
-    # For future use; can check if issuer matches /eauth\.va\.gov/ for SSOe
     def issuer
-      saml_response.issuers[0]
-    rescue
-      Raven.tags_context(controller_name: 'sessions', sign_in_method: 'not-signed-in:error')
-      raise
+      saml_response.issuer_text
```

Comment:
I'm a little concerned that we dropped the `rescue` which presumably was added for a reason.

Reply:
Oh, I see this is a new method and the rescue was copied from the authn_context method...I guess authn_context is checked early during parsing but would not be present on a SAML 'login failed' response. But I don't know if issuer would/would not be present or if we'd have the same issue if we read issuer on a long failed event.

Reply:
I guess worst case `issuer` would be nil and the `user_attribute_class` method would fall through.

Reply:
I'll add this back—I think the original intention behind this for error tracking within Sentry, since we re-raise the exception within the rescue